src/js/utils/__tests__/FormUtil-test.js

```diff
@@ -0,0 +1,79 @@
+jest.dontMock('../FormUtil');
+
+var FormUtil = require('../FormUtil');
+
+describe('FormUtil', function () {
+  describe('#getMultipleFields', function () {
+    describe('one property', function () {
+      beforeEach(function () {
+        this.result = FormUtil.getMultipleFields('uid', 2, {
+          name: 'uid'
+        });
+      });
+
+      it('returns the correct amount of fields', function () {
+        expect(this.result.length).toEqual(2);
+      });
+
+      it('returns the correct names for fields', function () {
+        expect(this.result[0].name).toEqual('uid[0]');
```

Comment:
well it would be confusing because if there are multiple props, it turns into an object.

it'll be:
`ports: [{name: xyz, password: secret}]`


rather than if it's just one prop it'll straight up be the value:
`ports: [100, string, something]`